tests/net/lib/tls_credentials/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  net.tls_credentials:
+    tags: net
```

Comment:
Could you add here "tls" tag, then it is easier to run just TLS tests.

Reply:
Done